AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_MaterialAdded.py

```diff
@@ -221,7 +221,7 @@ def AtomEditorComponents_Material_AddedToEntity():
         mesh_component.set_component_property_value(AtomComponentProperties.mesh('Mesh Asset'), model.id)
         general.idle_wait_frames(1)
         # Update mesh asset to a model with 5 LOD materials
-        model_path = os.path.join('Objects', 'sphere_5lods_fbx_psphere_base_1.azmodel')
+        model_path = os.path.join('Objects', 'sphere_5lods_fbx_psphere_base.azmodel')
```

Comment:
```suggestion
        model_path = os.path.join('Objects', 'sphere_5lods.azmodel')
```

Reply:
according to the test
"# Update mesh asset to a model with 5 LOD materials"
does sphere_5lods.azmodel have multiple LOD materials?

Reply:
sphere_5lods.azmodel as @amzn-tommy points to is likely correct. the material test will set lod materials that are not present on the simple sphere model. it expects 5 LOD's

Reply:
There is a model in testdata/objects/modelhotreload that satisfies the requirement for this test - a model with 5 lods. This comes from the TestData gem in Atom. These tests should use the one from TestData (my initial suggestion was misinformed), so that these tests can be more narrowly testing the functionality of assigning a model with multiple lods.

The one in the automated testing project, which is in objects/, is using a script at Editor/Scripts/auto_lod.py in its assetinfo. That deserves its own test focusing on validating the ScriptProcessorRule.

Reply:
so you're saying use this one https://github.com/o3de/o3de/blob/development/Gems/Atom/TestData/TestData/Objects/ModelHotReload/sphere_5lods.fbx
the path would need to update to 'Objects', 'ModelHotReload', 'sphere_5lods.fbx'

Reply:
The TestData model should have the same labels, and should work with the existing test.

Reply:
As discussed offline, the testdata asset is failing locally for some reason. We can follow up and make that change, and in the meantime this test should not be blocking dev from being green, so submitting the change that renames it from ..._base_1 to ..._base is a good interim solution.